### PR TITLE
New 'Who we are' that introduces North America

### PR DESCRIPTION
**Pull request by ElenaFdR**
The new website design spreads this content over 3 pages, so this is a short update to reflect our current reality.

### CODE REVIEW

**Review by ericherman — APPROVED** [review]

**Review by Ainali — APPROVED** [review]

File: who-we-are/index.md
Comment (brandmuse) [inline who-we-are/index.md:1]: Perhaps we could create a listable from this statement line 16 - _We do this by stewarding to public purpose, open source codebases, as well as sharing expertise, offering training and creating tools to make it easier for public organizations to build and maintain software together._- so it's easier for search engine optimization. 

-  Manage open-source codebases for public purpose projects.
- Share our industry-leading expertise.
-  Offer tailored training for public organizations.
-  Develop user-friendly software tools.
- Facilitate collaborative software development and maintenance for public entities.

Comment (ElenaFdR) [reply who-we-are/index.md:1]: I've added all but the first bullet - we're usually careful to say we don't manage codebases, as we insist on codebases having other maintainers so we can be more effective, neutral facilitators. 
Take a look now? I have a concern that the first and last bullets are duplicates, but aimed at different audiences (ie, people familiar with open source codebases will understand stewardship; people with a more mainstream/policy background will better understand the last point).

**Review by ericherman — APPROVED** [review]

File: who-we-are/index.md [hunk 1]
```diff
@@ -11,11 +11,19 @@ redirect_from:
...
+We do this by:
 
-Based on their experience of technological innovation in a traditional industry, Pieter van der Does and Arnout Schuijff (co-founders of the Adyen payments platform) agreed with Ben and Boris that public organizations deserved better software. They joined the Board of Directors to share their expertise and connections.
+* stewarding public purpose, open source codebases
+* sharing our industry-leading expertise
+* offering tailored training for public organizations
+* developing tools to make it easier for public organizations to build and maintain software together
+* facilitating collaborative software development and maintenance for public organizations
+
+Ben Cerveny, Boris van Hoytema and Arnout Schuijff founded the Foundation in 2019 in Amsterdam, based on their realization that stewardship was the key to converting the massive public investment in open source and collaboration into long-term, sustainable implementation.
```
Comment (Ainali) [inline who-we-are/index.md:24]:
```suggestion
Ben Cerveny, Boris van Hoytema and Arnout Schuijff founded the Foundation in 2019, based on their realization that stewardship was the key to converting the massive public investment in open source and collaboration into long-term, sustainable implementation. We were founded  in Amsterdam, with and aim to operate globally. 
```
Just a suggestion to clarify both whose realization it was (not gemeente Amsterdam) and the ambition.

**Review by Ainali — APPROVED** [review]

File: who-we-are/index.md
Comment (brandmuse) [reply who-we-are/index.md:1]: I can see that. Great!

Comment (brandmuse) [reply who-we-are/index.md:1]: The other minor edit I'd make is the sentence: In 2022, _we launched the Foundation for Public Code North America, to support growing interest in collaborative software **there**._ I'd change THERE to IN THAT REGION.

Comment (ElenaFdR) [reply who-we-are/index.md:1]: Done!

**Pull request merged**